executorlib/task_scheduler/interactive/blockallocation.py

```
@@ -61,11 +61,16 @@ def __init__(
         self._process_kwargs = executor_kwargs
         self._max_workers = max_workers
+        self._shutdown_flag = False
         self._set_process(
             process=[
                 Thread(
                     target=execute_tasks,
-                    kwargs=executor_kwargs | {"worker_id": worker_id},
+                    kwargs=executor_kwargs
+                    | {
+                        "worker_id": worker_id,
+                        # "stop_function": lambda: self._shutdown_flag,
```

Ruff (0.12.2)

Found commented-out code

Remove commented-out code

(ERA001)